app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated.")
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Visitor's vaccine is outdated.")
+        elif not visitor.get("wearing_a_mask", False):
+            raise NotWearingMaskError("Visitor is not wearing a mask.")
+        else:
+            return f"Welcome to {self.name}"
```

Comment:
According to the checklist, avoid using unnecessary `else` after `elif` statements. You can directly return the welcome message without the `else` block, as the previous conditions already handle the exceptions.